features/api_services_list.feature

```diff
@@ -0,0 +1,184 @@
+Feature: Services list api
+
+  @uses.config.contract_token
+  Scenario Outline: Get services list when unattached/attached as non-root
+    Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
+    When I verify root and non-root `pro api u.pro.services.list.v1` calls have the same output
+    When I run `pro api u.pro.services.list.v1` as non-root
+    Then stdout is a json matching the `api_response` schema
```

Comment:
Shouldn't we be using `api_service_list_response` here ?

Reply:
Fixed, also checking the correct schema now